src/Sdk/Sdk.csproj

```diff
@@ -17,7 +17,7 @@
         <PackageReference Include="Microsoft.Win32.Registry" Version="4.4.0" />
         <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
         <PackageReference Include="Microsoft.AspNet.WebApi.Client" Version="5.2.9" />
-        <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="5.2.1" />
+        <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="7.2.0" />
```

Comment:
What is a policy on choosing a version for the deps? https://github.com/advisories/GHSA-59j7-ghrg-fj52 proposes 3 different patches depends on the interval:

| Affected versions | Patched versions |
|---|---|
|< 5.7.0 | 5.7.0 |
| >= 6.5.0, < 6.34.0 | 6.34.0 |
| >= 7.0.0-preview, < 7.1.2 | 7.1.2 |